apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.DatabaseStatistics.cs

```diff
@@ -14,95 +14,87 @@
 
 
 using Google.Cloud.Spanner.Common.V1;
-
+using System;
+
 namespace Google.Cloud.Spanner.V1
 {
     public partial class SessionPool
     {
         /// <summary>
-        /// A snapshot of statistics for one database within a <see cref="SessionPool"/>.
+        /// A snapshot of statistics for a <see cref="SessionPoolSegmentKey"/> within a <see cref="SessionPool"/>.
         /// </summary>
+        [Obsolete("Use Google.Cloud.Spanner.V1.SessionPool.SessionPoolSegmentStatistics instead. The behavior of both classes is the same, but the new class name better reflects the fact that sessions are now pooled by aspects other than database name.")]
         public sealed class DatabaseStatistics
         {
             /// <summary>
             /// The database this set of statistics corresponds to.
             /// </summary>
-            public DatabaseName DatabaseName { get; }
+            public DatabaseName DatabaseName => PoolSegmentStatistics.DatabaseName;
+
+            /// <summary>
+            /// The database role of the sessions in the session pool this set of statistics corresponds to. May be null
+            /// in which case the default (public) database role is used if fine grained access control
+            /// is enabled; otherwise the database role is ignored.
+            /// </summary>
+            public string DatabaseRole => PoolSegmentStatistics.DatabaseRole;
+
+            private SessionPoolSegmentStatistics PoolSegmentStatistics  { get; }
```

Comment:
I'd just call this SegmentStatistics. ("Session pool" feels like a single concept here which we shouldn't just have half of. I think it's okay for that to be implicit.)